Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/storage/adlsgen2/ADLSGen2FSOperation.java

```diff
@@ -78,15 +81,21 @@ public ADLSGen2FSOperation(@NotNull HttpObservable http) {
                 .setAction("flush");
     }
 
-    public Observable<Boolean> createDir(String dirpath) {
```

Comment:
It might be better to keep this method:
`public Observable<Boolean> createDir(String dirpath)` and add  a `public Observable<Boolean> createDir(String dirpath, String permission)` version with a default `uMask` value,